inline-demo.html

```diff
@@ -47,6 +47,9 @@
   var demoId;
 
   function receiveMessage(event) {
+    if (!event.origin.match(/^https:\/\/\w*\.?webcomponents\.org/))
```

Comment:
It's not really a concern in this case. The iframe is already executing on an origin with user content.